docs/application/tizen-studio/rt-ide/getting-started/install.md

```diff
@@ -1,45 +1,67 @@
-# Installing Tizen Studio for RT
+# Install Tizen Studio for RT IDE
 
-To install the Tizen Studio for RT:
+This section explains about the prerequisites required to install Tizen Studio for RT IDE. It also walks you through the installation steps.
 
-1. Accept the software license.
+## Prerequisites
 
-   The license contains important legal notices for using the Tizen Studio. Read the complete agreement, and click **Accept** only if you agree with the license statement:
+Before installing Tizen Studio for RT IDE, ensure that the following prerequisites are met:
 
-   ![Software license agreement](./media/rt_install_license.png)
+1. Download the appropriate JDK version from the [official Oracle website](https://www.oracle.com/technetwork/java/javase/downloads/index.html).
```

Comment:
Global: Please ignore the alignment of the first gif. We are working on fixing it.